### PR TITLE
applications: nrf_desktop: Update USB state documentation

### PR DESCRIPTION
**Pull request by MarekPieta**
Change updates documentation for the USB state module.

Jira: NCSDK-17315

### CODE REVIEW

File: applications/nrf_desktop/doc/usb_state.rst [hunk 1]
```diff
@@ -38,25 +38,26 @@ Additionally, you can also configure the options described in the following sect
 Low latency device configuration
 ================================
 
-For low latency devices, make sure that the device requests a polling rate of 1 ms by setting :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` to ``1``.
+The module sets default value of :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` to ``1``.
+Requesting polling rate of 1 ms is done to ensure low latency of HID reports.
```
Comment (divipillai) [inline applications/nrf_desktop/doc/usb_state.rst:42]:
```suggestion
For low latency of HID reports, the device requests a polling rate of 1 ms.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -190,6 +190,9 @@ nRF Desktop
 -----------
 
 * The UUID16 values of GATT Human Interface Device Service (HIDS) and GATT Battery Service (BAS) are moved from advertising data to scan response data.
+* Introduced :ref:`CONFIG_DESKTOP_USB_REMOTE_WAKEUP <config_desktop_app_options>` Kconfig option for :ref:`nrf_desktop_usb_state`.
+  The opion enables the USB wakeup functionality in the application.
```
Comment (divipillai) [inline doc/nrf/releases/release-notes-changelog.rst:194]:
```suggestion
  The option enables the USB wakeup functionality in the application.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -194,6 +194,9 @@ nRF Desktop
 -----------
 
 * The UUID16 values of GATT Human Interface Device Service (HIDS) and GATT Battery Service (BAS) are moved from advertising data to scan response data.
+* Introduced :ref:`CONFIG_DESKTOP_USB_REMOTE_WAKEUP <config_desktop_app_options>` Kconfig option for :ref:`nrf_desktop_usb_state`.
+  The option enables the USB wakeup functionality in the application.
+  The option selects :kconfig:option:`CONFIG_USB_DEVICE_REMOTE_WAKEUP`.
```
Comment (kapi-no) [inline doc/nrf/releases/release-notes-changelog.rst:199]: I think you can leave out this sentence. The release notes shouldn't contain too many details about the feature

**Review by kapi-no — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 3 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:199]: This changes API, so maybe we could leave it here to make sure users would update their configurations ?

Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:199]: If users enable the functionality only in the stack, it would silently fail.

**Review by pdunaj — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Pull request merged**